Separate remove switch to function (#6)
Stealthbits/ServerUntrustAccount: README.md

  AES128: j916...
  AES256: k093...
```

## Remove-ServerUntrustAccount

> This function is a clean up function to remove persistence items created from Add-ServerUntrustAccount. It performs the following activities:

Reviewer: ```suggestion
This function removes accounts and permissions created using `Add-ServerUntrustAccount`. It performs the following actions:
```

>  * Remove ACE for the backdoor security principal on Domain with DS-Install-Replica permission

Reviewer: ```suggestion
 * Remove the DS-Install-Replica permission on the domain object for the specified computer or managed service account
```

>  * Remove ACE on the computer / managed service account object to modify UserAccountcontrol attribute

Reviewer: ```suggestion
 * Remove the modify userAccountControl permission on the specified computer or managed service account object
```

>  * When deleteComputer parameter is specified, also delete the computer / managed service account object

Reviewer: ```suggestion
 * With the deleteComputer switch enabled, also deletes the specified computer or managed service account object
```
